src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -575,6 +547,93 @@ public override void VisitMethod(IMethodSymbol symbol)
                 AddParameters(symbol);
                 AddTypeParameterConstraints(symbol);
             }
+
+            void visitFunctionPointerSignature(IMethodSymbol symbol)
```

Comment:
```suggestion
            void VisitFunctionPointerSignature(IMethodSymbol symbol)
```

Reply:
This is not the pattern in the compiler.

---
In reply to: [463843297](https://github.com/dotnet/roslyn/pull/46467#discussion_r463843297) [](ancestors = 463843297)